src/services/administration/iam/policy/PolicyPage.vue

```diff
@@ -30,25 +30,12 @@ import { useManagePermissionState } from '@/common/composables/page-manage-permi
 
 import PolicyListDataTable from '@/services/administration/modules/PolicyListDataTable.vue';
 import { ADMINISTRATION_ROUTE } from '@/services/administration/route-config';
-import { administrationStore } from '@/services/administration/store';
+import { usePolicyStore } from '@/services/administration/store/policy-page-store';
 
-export default {
-    name: 'PolicyPage',
-    components: {
-        PHeading,
-        PButton,
-        PolicyListDataTable,
-    },
-    setup() {
-        const state = reactive({
-            totalCount: computed(() => administrationStore.state.policy.totalCount),
-            hasManagePermission: useManagePermissionState(),
-        });
+const policyStore = usePolicyStore();
 
-        return {
-            ...toRefs(state),
-            ADMINISTRATION_ROUTE,
-        };
-    },
-};
+const state = reactive({
+    totalCount: computed(() => policyStore.state.totalCount),
```

Comment:
Question: `computed` is required? Isn't `policyStore.state` a responsive state? 🤔

Reply:
That would be better!